src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <VcardTemplate id="extension-request">
+    <template #icon>
+      mdi-calendar-range
+    </template>
+
+    <template #title>
+      Extension Request
+    </template>
+
+    <template #content>
+      <div class="px-6 py-7">
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>Is this the first AGM?</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <v-radio-group
+              v-model="data.isFirstAgm"
```

Comment:
The v-model is a 2-way binding.

Since `data` was passed by reference, updating "isFirstAgm" here updates it in the data object.